tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -128,11 +204,11 @@ private string GetImage(string imageType, string version, string osVariant)
             // Ensure image exists locally
             if (IsLocalRun)
             {
-                Assert.True(DockerHelper.ImageExists(image), $"`{image}` could not be found on disk.");
+                Assert.True(_dockerHelper.ContainerExists(image), $"`{image}` could not be found on disk.");
```

Comment:
This should still be calling `ImageExists`.

Reply:
I updated this so that PR validation would hopefully be complete by time Hong gets in.

Reply:
My local run didn't reach this code path.